modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -1089,39 +1088,52 @@ protected function content_template() {
 			var elementUid = view.getIDInt().toString().substr( 0, 3 ); #>
 			<div class="e-n-tabs-heading" role="tablist">
 				<# _.each( settings['tabs'], function( item, index ) {
-				var tabCount = index + 1,
-				tabUid = elementUid + tabCount,
-				tabTitleKey = 'tab-title-' + tabUid;
-				tabIcon = elementor.helpers.renderIcon( view, item.tab_icon, { 'aria-hidden': true }, 'i' , 'object' );
+					let tabCount = index + 1,
```

Comment:
I had to change the icon display to the change the icon display behaviour.

Reply:
Is the display behaviour is changed only in the editor?

Reply:
@davseve I have changed it in the php as well, see: https://github.com/elementor/elementor/blob/efa4c5a86fbe567424de046f947222ed5645da51/modules/nested-tabs/widgets/nested-tabs.php#L1061.